app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -64,6 +64,26 @@ class DradisDatatable {
       },
       lengthChange: false,
       pageLength: 25,
+      stateSave: true,
+      stateSaveCallback: function(settings, data) {
+        var dataTableDomId = settings.oInstance[0].id;
+        var localStorageKey = `Datatables_${dataTableDomId}_${window.location.pathname}`;
```

Comment:
This looks like a variable we can save as a class variable, then accessible with `this.localStorageKey`

Reply:
Opted to use data-local-storage-key="key" and saved it as a the object's attribute